js/game.js

```diff
@@ -1149,16 +1149,18 @@ function init_socket(args)
 		window.socket.destroy();
 	}
 	$(".disconnected").hide();
-	if(is_sdk && (Cookies.get("windows") || Cookies.get("local_ip") || window.location.host=="advanture.land" || window.location.host=="x.thegame.com")) server_addr="192.168.1.125"; // Cookies.set('windows','1',{expires:12*365});
-	else if(is_sdk)
-	{
-		if(window.location.origin=='http://127.0.0.1/') server_addr="127.0.0.1";
-		else server_addr="0.0.0.0";
-	}
+	// if(is_sdk && (Cookies.get("windows") || Cookies.get("local_ip") || window.location.host=="advanture.land" || window.location.host=="x.thegame.com")) server_addr="192.168.1.125"; // Cookies.set('windows','1',{expires:12*365});
+	// else if(is_sdk)
+	// {
+	// 	server_addr = window.location.hostname
+	// 	console.log('SDK MODE wss server adress:', location.protocol, server_addr, server_port)
+	// }
```

Comment:
Can you tell me why this was commented, please?  
Moreover, it seems like the commented code is not exactly what was there before, this can be confusing later.

If the code is not needed anymore, remove it entirely instead, please

Reply:
It was commented out because the mapping in environment now sets the correct ip in the server persisted in appserver and later returned by the characters_and_servers api request that the client uses to establish the web socket connection to the game server

It should be removed yeah.

Reply:
Commented out code has been removed